modules/vf-graphql-holochain/resolvers/plan.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Resolvers for EconomicEvent fields
+ *
+ * @package: HoloREA
+ * @since:   2019-08-27
+ */
+
+import { DNAIdMappings, DEFAULT_VF_MODULES, VfModule } from '../types'
+import { extractEdges, mapZomeFn } from '../connection'
+
+import {
+  Process,
+  PlanProcessFilterParams
+} from '@valueflows/vf-graphql'
+
+
+export default (enabledVFModules: VfModule[] = DEFAULT_VF_MODULES, dnaConfig: DNAIdMappings, conductorUri: string) => {
+
+  const hasAgent = -1 !== enabledVFModules.indexOf(VfModule.Agent)
+  const hasKnowledge = -1 !== enabledVFModules.indexOf(VfModule.Knowledge)
+  const hasPlanning = -1 !== enabledVFModules.indexOf(VfModule.Planning)
+  const hasAgreement = -1 !== enabledVFModules.indexOf(VfModule.Agreement)
+
+  const readProcesses = mapZomeFn(dnaConfig, conductorUri, 'observation', 'process_index', 'query_processes')
+
+  return Object.assign(
+    (hasPlanning ? {
+      processes: async (filter: PlanProcessFilterParams): Promise<Process[]> => {
```

Comment:
here I was trying to do something similar to the `fulfillment` field of `EconomicEvent`: https://lab.allmende.io/valueflows/vf-schemas/vf-graphql/-/blob/sprout/lib/schemas/bridging/observation.planning.gql
which I am basing off of: https://lab.allmende.io/valueflows/vf-schemas/vf-graphql/-/blob/sprout/lib/schemas/bridging/observation.plan.gql

Reply:
Yep, you're onto it. `Process` currently lives in the "observation" DNA however, so make it a `hasObservation` & appropriate check (: